finals-2024-01-15/code/5105-angela-cekovska/src/app/about/about.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+
+@Component({
```

Comment:
+4 for the about component - it's not available via routing, and there's no actual student name 😃